### PR TITLE
fix(search): only temporal attributes must be retrieved #238

### PR DESCRIPTION
**Pull request by philippestoltz**

### CODE REVIEW

File: search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityHandler.kt [hunk 1]
```diff
@@ -160,20 +163,16 @@ internal fun buildTemporalQuery(params: MultiValueMap<String, String>): Temporal
     if (timerelParam == "between" && endTimeParam == null)
         throw BadRequestDataException("'endTime' request parameter is mandatory if 'timerel' is 'between'")
 
-    val (timerel, time) = if (timerelParam == null && timeParam == null) {
```
Comment (bobeal) [inline search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityHandler.kt:163]: Not sure the code is easier to read and maintain in this new version...

Comment (philippestoltz) [reply search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityHandler.kt:163]: it should be better now

File: shared/src/main/kotlin/com/egm/stellio/shared/util/ApiUtils.kt [hunk 2]
```diff
@@ -15,11 +18,11 @@ object ApiUtils {
     }
 }
 
-fun String.parseTimeParameter(errorMsg: String): ZonedDateTime =
+fun String.parseTimeParameterwithEither(errorMsg: String): Either<String, ZonedDateTime> =
```
Comment (HoucemKacem) [inline shared/src/main/kotlin/com/egm/stellio/shared/util/ApiUtils.kt:21]: Maybe you can keep the old function name parseTimeParameter

**Review by HoucemKacem — APPROVED** [review]

File: shared/src/main/kotlin/com/egm/stellio/shared/util/ApiUtils.kt (hunk 2 above)
Comment (philippestoltz) [reply shared/src/main/kotlin/com/egm/stellio/shared/util/ApiUtils.kt:21]: OK

**Review by bobeal — APPROVED** [review]

**Pull request merged**